GameData/RealismOverhaul/Engine_Configs/R103_Config.cfg

```diff
@@ -0,0 +1,155 @@
+//	==================================================
+//	R-103
+//
+//	Manufacturer: Kostin Bureau
+//
+//	=================================================================================
+//	R-103 (2.07 x 0.1 m)
+//	MR-1 Booster (cluster of 8)
+//
+//	Dry Mass: 8 kg					//literally a steel tube filled with solid fuel (24 kg full - 1 kg warhead - 11.6 kg fuel)
+//	Dimensions: 1.81 x 0.1 m 		(without warhead, tank lenght 1.44m - 69.5% of total 2.07 m usable)
+//	Thrust (SL): 10.175 kN
+//	Thrust (Vac): 11.192 kN
+//	ISP: 200 SL / 220 Vac
+//	Burn Time: 2.92 s
+//	Chamber Pressure: 5.2689 MPa	(52 atm)
+//	Propellant: solid
+//	Prop Ratio: 4.0 (liquid)
+//	Throttle: N/A
+//	Nozzle Ratio: ???
+//	Ignitions: 1
+//	=================================================================================
+//	Sources:	http://www.astronautix.com/r/r-103.html
+//				https://second.wiki/wiki/r-103_misil
+//				http://www.astronautix.com/r/r-110.html
+//				https://ru.wikipedia.org/wiki/Тайфун_(зенитная_ракета)
+//				https://en.wikipedia.org/wiki/Taifun_(rocket)
+//				https://www.airplanesandrockets.com/rockets/soviet-mr-1-meteo-rocket-march-1967-american-modeler.htm
+//	
+//	Used by:
+//	Notes:
+//	propellant load 10.25 kg
+//	==================================================
+@PART[*]:HAS[#engineType[R103]]:FOR[RealismOverhaulEngines]
+{
+	%title = R-103
+	%manufacturer = Kostin Bureau
+	%description = Post-war Russian version of German Taifun anti-aircraft barrage rocket. Developed and tested in 1947-1951 but abandoned in favor of the R-110. A cluster of 8 of these rockets formed the first stage of the MR-1 sounding rocket. Length without payload - 1.81 meters. 
+	
+	@MODULE[ModuleEngines*]
+	{
+		%EngineType = SolidBooster
+		%allowShutdown = False
+		%throttleLocked = True
+		%autoVariationScale = 3
+	}
+	
+	!MODULE[ModuleGimbal] {}
+	!MODULE[ModuleFuelTanks],* {}
+	!MODULE[ModuleAlternator],*{}
+	!RESOURCE,*{}
+	
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 6.41
+		type = NGNC
+		basemass = -1
+		TANK
+		{
+			name = NGNC
+			amount = 6.41
+			maxAmount = 6.41
+		}
+	}
+	
+	MODULE
+	{
+		name = ModuleEngineConfigs
+		type = ModuleEngines
```

Comment:
Is it just "ModuleEngines" in the template or should this be ModuleEnginesRF?
Note there's a patch in RO's global config that will change all ME to MERF and all references in the MEC to MERF.  So the end result is MERF, and this question is more about consistency.

Reply:
Every single engine config is set up this way. It's probably meant to be that way.